src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  background: $color-bg-white;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-right: $card-margin-right;
+  margin-bottom: 46px;
```

Comment:
do not use external styles on BEM block
use [mixes](https://en.bem.info/forum/4/) instead